developer-docs-site/docs/guides/move-guides/move-scripts.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Move Scripts"
+slug: "move-scripts"
+---
+
+# Move Scripts
+
+This tutorial explains how to write and execute a Move script. You can use Move scripts to execute an arbitrary piece of code as the transaction payload.
+
+## Example use case
+
+The following example calls functions on the [aptos_coin.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/aptos_coin.move) module to confirm the balance of the destination account is less than `desired_balance`, and if so, tops it up to `desired_balance`.
+
+```move
+script {
+    use std::signer;
+    use aptos_framework::aptos_account;
+    use aptos_framework::aptos_coin;
+    use aptos_framework::coin;
+
+    fun main(src: &signer, dest: address, desired_balance: u64) {
+        let src_addr = signer::address_of(src);
+
+        let balance = coin::balance<aptos_coin::AptosCoin>(src_addr);
+        if (balance < desired_balance) {
+            aptos_account::transfer(src, dest, desired_balance - balance);
```

Comment:
nit: Might as well demonstrate two calls, especially those against public fun functions that are not callable as normal transactions. For example:

let withdraw_coins = aptos_account::withdraw_coins<AptosCoin>(...);
aptos_account::deposit_coins(dest, withdraw_coins);

Reply:
ya sure why not